### DISCUSSION TITLE
Schizo artwork?

### DISCUSSION THREAD

**Discussion by shitbread** [hongbo-miao/hongbomiao.com#20431] [Q&A] [answered]
Stumbled upon this from stackoverflow. Is this repo basically homelab schizo artwork? I don't get it.

**Comment by hongbo-miao** [comment, marked as answer]
Hi @shitbread! This is just a place for me to get hands-on experience after taking [some courses](https://www.credential.net/profile/hongbo-miao/wallet). If I find a technology to be of good quality and potentially useful in certain situations, I might add it here. It also serves as a way to share knowledge. Hope it clarifies! 😉
